pkg/alias/gateway/gateway.go

```diff
@@ -0,0 +1,29 @@
+package gateway
+
+import (
+	"github.com/TykTechnologies/tyk/gateway"
+	"github.com/TykTechnologies/tyk/internal/event"
+)
+
+const (
+	EventBreakerTriggered  = event.BreakerTriggered
+	EventRateLimitExceeded = event.RateLimitExceeded
+	EH_WebHook             = event.WebHookHandler
```

Comment:
why not `EventWebHookHandler` ? or in such case at least remove the underscore

Reply:
added the suggestion, and renamed `EH_Webhook` usage in dashboard